src/sdk/wxscintilla/include/wx/wxscintilla.h

```diff
@@ -44,6 +44,18 @@
 #include <wx/versioninfo.h>
 #endif
 
+// this workarounds an issue that in some build configuration of wx library, WXEXPORT is defined as empty,
+// as WXDLLIMPEXP_SCI is defined as WXEXPORT below, this may cause wxScintilla class not be exported from
+// the codeblocks.dll
+#ifdef __WXMSW__
```

Comment:
This is totally wrong. Don't define `WXEXPORT`. It might affect other headers globally. It is better to change how `WXDLLIMPEXP_SCI` is defined below.